packages/uplosi.conf.template

```diff
@@ -0,0 +1,11 @@
+[base]
+imageVersion = "0.0.1"
```

Comment:
We need a mechanism to update the version on each upload. uplosi has separate image file and -i flag for that.

Reply:
I think we need a mechanism to detect whether an image upload is needed or not. I thought we'd to this via version, but there might be other optioins.

Reply:
I suppose we could create a file with a name based on `basename $(readlink result)` (e.g. `ay580z91d48zd0irvzfzjph8k4q07sd4-image-podvm-gpu-1-rc1`) and place it `workspace.cache` after uploading. On subsequent runs, we could check if the file already exists and skip the upload.

Reply:
We could put the image id of the uploaded image in that file.